Zephyr/arm_toolchain.cmake.in

```diff
@@ -0,0 +1,32 @@
+include(CMakeForceCompiler)
+
+set(CMAKE_SYSTEM_NAME Generic)
+set(CMAKE_SYSTEM_PROCESSOR arm)
+set(CMAKE_CROSSCOMPILING 1)
+set(CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)
+set(PLATFORM_NAME "Zephyr")
+
+set(CMAKE_SYSROOT @CMAKE_SYSROOT@)
+
+set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
+set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY)
+set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
+
+# Makefile flags
+#set(CROSSDEV @CROSS_COMPILE_PREFIX@)
+#set(ARCH_CPU_FLAGS @ARCH_CPU_FLAGS@)
+#set(ARCH_OPT_FLAGS @ARCH_OPT_FLAGS@)
```

Comment:
Could you remote it?

Reply:
Please let me keep them as comments until we finish the work on the Zephyr example because I'm not sure right now if they are necessary or not. In FreeRTOS case we use these flags... I'll remove later if they are not necessary.